content/es/contribute/_index.md

```diff
@@ -123,6 +125,9 @@ Copie el contenido…
 ![copiar contenido](/images/how-to/howto-08.png)
 
 Cambie de branch a "dev-es", regrese a la carpeta /content/ y seleccione "es". 
+
+![cambiar de branch](/images/how-to/switch-branch.png)
```

Comment:
Of course this image can be used as a general example for many localization teams,
but we might move to e.g., `content/es/contribute/images/switch-branch.png`
then other localization team might add their own screenshots too.

Reply:
@jihoon-seo I think the path of the customized image needs to be relative to the `static` folder, in other words, maybe something like `static/images/content/es/` right? So then the reference can as follows:
```suggestion
![cambiar de branch](/images/content/es/switch-branch.png)
```